### PR TITLE
Broke redirect loop between kong and cloudflare. Fixed CORS issue

### PR DESCRIPTION
**Pull request by kurtisassad**
## Description
Change it so that if the request to the server comes from kong, we don't forward it to the reverse proxy.

Also fixed CORS issue with swagger docs by sending directly to kong, and then on kongs side returning the correct Access-Control-Allow-Origin with the kong CORS plugin

## Motivation and Context
The old way was causing a redirect loop between kong and the reverse proxy.

## Does this PR affect any server routes?
chose YES or NO

## Have you added the issue number here?
(In the right sidebar, under "development")

### CODE REVIEW

**Review by dillchen — APPROVED** [review]
LGTM, @MuonShot and @jnaviask for merge plan and or QA on staging2

File: packages/commonwealth/server/routing/external.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
+import { app } from 'chain-events/services/app/Server';
```
Comment (jnaviask) [inline packages/commonwealth/server/routing/external.ts:1]: What is this import?

File: packages/commonwealth/server.ts [hunk 2]
```diff
@@ -125,9 +125,13 @@ async function main() {
   const setupMiddleware = () => {
     // redirect from commonwealthapp.herokuapp.com to commonwealth.im
     app.all(/.*/, (req, res, next) => {
+      if (req.ip.includes(process.env.DISABLE_REDIRECT_URL)) {
```
Comment (jnaviask) [inline packages/commonwealth/server.ts:128]: We need safe defaults for these, in `config.ts`.

**Review by jnaviask — CHANGES_REQUESTED** [review]
Some small changes...

File: packages/commonwealth/server.ts (hunk 2 above)
Comment (kurtisassad) [reply packages/commonwealth/server.ts:128]: Safe default should be undefined (which is what process.env.DISABLE_REDIRECT_URL would resolve to).

Unless you are thinking of some other default we should have for this?

File: packages/commonwealth/server/routing/external.ts (hunk 1 above)
Comment (kurtisassad) [reply packages/commonwealth/server/routing/external.ts:1]: Thanks, missed this

File: packages/commonwealth/server.ts (hunk 2 above)
Comment (jnaviask) [reply packages/commonwealth/server.ts:128]: Undefined is ok for this one (unless `.includes(undefined)` acts weird?), but what about for the other two? They'll only be set on heroku.

**Comment by CowMuon** [comment]
What's required here so local development setups don't break? @kurtisassad

**Review by jnaviask — APPROVED** [review]

**Review by CowMuon — APPROVED** [review]

**Pull request merged**